src/Compilers/Core/Portable/Syntax/InternalSyntax/SyntaxListBuilder.cs

```diff
@@ -156,9 +158,9 @@ public bool Any(int kind)
             return false;
         }
 
-        public GreenNode[] ToArray()
+        public GreenNode?[] ToArray()
```

Comment:
❔ Shouldn't this return `GreenNode[]`? The current implementation of this method is not consistent with the suppression added in the current implementation of `Any`.

Reply:
Guess we can make this one a stronger contract.

---
In reply to: [363549234](https://github.com/dotnet/roslyn/pull/40767#discussion_r363549234) [](ancestors = 363549234)